src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      data-qa="big"
+      method="GET"
+      class="field"
+    >
+      <div>
+        <img
+          src="images/Search.svg"
+          alt="search"
+          class="big-icon"
+        >
+        <input
+          type="search"
+          class="first-search"
```

Comment:
Почему first?
Если их местами в доме поменять - классы тоже надо будет менять?
```suggestion
          class="search"
```